internal/cmd/testdata/edgecase/generator_config.yml

```diff
@@ -23,6 +30,11 @@ data_sources:
     read:
       path: /nested_collections
       method: GET
+    schema:
+      ignores:
+        - triple_nested_map.ignore_me_1
+        - triple_nested_map.obj_prop.ignore_me_2
```

Comment:
Open to a bikeshed on naming/location of the config. I chose to keep it outside of the `schema.attributes` section because it happens _before_ mapping, whereas overriding happens _after_ mapping.